### PR TITLE
Feature/dx 887 git commit hooks

### PR DESCRIPTION
**Pull request by NullableInt**
This makes available two hooks that validates the name of your branch, and that your commit contains the task number of your branch.

### CODE REVIEW

File: .docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM jekyll/jekyll:4.0.0
```
Comment (asbjornu) [inline .docker/Dockerfile:1]: Should we perhaps drop this to `3.8.6` here as well?

```suggestion
FROM jekyll/jekyll:3.8.6
```

File: _includes/alert-callback-url.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 {% assign callback_href = "/payments/" | append: payment_instrument | append: "/other-features#callback" %}
 
 {% capture body %}
-While optional, it is strongly recommended to set a [`callbackUrl`]({{
+It is mandatory to set a [`callbackUrl`]({{
```
Comment (asbjornu) [inline _includes/alert-callback-url.md:5]: What's this doing in here?

File: payments/invoice/after-payment.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ sidebar:
       title: Seamless View
     - url: /payments/invoice/direct
       title: Direct
+    - url: /payments/invoice/capture
```
Comment (asbjornu) [inline payments/invoice/after-payment.md:15]: Hm?

File: _includes/alert-callback-url.md (hunk 2 above)
Comment (NullableInt) [reply _includes/alert-callback-url.md]: Wasn't updated from dev.

File: payments/invoice/after-payment.md (hunk 3 above)
Comment (NullableInt) [reply payments/invoice/after-payment.md]: Wasn't updated from dev.

File: .docker/Dockerfile (hunk 1 above)
Comment (NullableInt) [reply .docker/Dockerfile]: It's either that, or wait for our own image and drop that all together.

**Pull request merged**